packages/session-replay-browser/src/helpers.ts

```diff
@@ -130,3 +137,17 @@ export const getServerUrl = (serverZone?: keyof typeof ServerZone): string => {
 
   return SESSION_REPLAY_SERVER_URL;
 };
+
+export const getStorageSize = async (): Promise<StorageData> => {
+  const globalScope = getGlobalScope();
+  if (globalScope) {
+    const storeEstimate: ChromeStorageEstimate = await globalScope.navigator.storage.estimate();
+    const totalStorageSize = storeEstimate.usage ? Math.round(storeEstimate.usage / KB_SIZE) : 0;
+    const percentOfQuota =
+      storeEstimate.usage && storeEstimate.quota
+        ? Math.round((storeEstimate.usage / storeEstimate.quota + Number.EPSILON) * 1000) / 1000
```

Comment:
Why the use of `Number.EPSILON` here?

Reply:
It's a surprisingly controversial way of helping JS to round a number with floating points: https://stackoverflow.com/questions/11832914/how-to-round-to-at-most-2-decimal-places-if-necessary